test/parallel/test-event-emitter-check-listener-leaks.js

```diff
@@ -13,6 +13,14 @@ assert.ok(!e._events['default'].hasOwnProperty('warned'));
 e.on('default', function() {});
 assert.ok(e._events['default'].warned);
 
+// symbol
+var sym = Symbol('x');
```

Comment:
We are in the process of migrating the code from `var` to `const`. It's not really practical to update a codebase this large in one pass, so we make the updates as we touch the code. Can you please add new code using `const`.